01week/rockPaperScissors.js

```diff
@@ -9,9 +9,25 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
   // Write code here
-
+  // function should take in two inputs and then compare them to see which hand wins a game of rock, paper, scissors
+  //normalize the inputs to remove errors arising from lower/uppercase letters and spaces
+  //use .trim() and .toLowerCase()
+  hand1 = hand1.toLowerCase().trim();
```

Comment:
In functional programming you want to avoid premaritally changing given parameters. A better practice would be to create new variables.
```javascript
    const  hand1Formatted = hand1.toLowerCase().trim();
```